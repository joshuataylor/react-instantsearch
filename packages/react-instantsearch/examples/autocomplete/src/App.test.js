import React from 'react';
import MultiIndex from './App-Multi-Index';
import Mentions from './App-Mentions';
import renderer from 'react-test-renderer';

jest.mock('antd/lib/mention');

describe('autocomplete recipe3', () => {
  it('MultiIndex renders without crashing', () => {
    const component = renderer.create(<MultiIndex />);

    expect(component.toJSON()).toMatchSnapshot();
  });
  it('Mentions renders without crashing', () => {
    const component = renderer.create(<Mentions />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
