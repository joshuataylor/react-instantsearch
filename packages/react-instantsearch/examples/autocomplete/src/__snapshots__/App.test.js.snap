// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`autocomplete recipe Mentions renders without crashing 1`] = `
<div
  className="ais-InstantSearch__root"
/>
`;

exports[`autocomplete recipe Mentions renders without crashing v 1`] = `
<div
  className="ais-InstantSearch__root"
/>
`;

exports[`autocomplete recipe MultiIndex renders without crashing 1`] = `
<div
  className="ais-InstantSearch__root"
>
  <div
    className="react-autosuggest__container"
  >
    <input
      aria-activedescendant={null}
      aria-autocomplete="list"
      aria-expanded={false}
      aria-haspopup={false}
      aria-owns="react-autowhatever-1"
      autoComplete="off"
      className="react-autosuggest__input"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      placeholder="Type a product"
      role="combobox"
      type="text"
      value=""
    />
    <div
      className="react-autosuggest__suggestions-container"
      id="react-autowhatever-1"
    />
  </div>
</div>
`;

exports[`autocomplete recipe autocomplete recipe Mentions renders without crashing 1`] = `
<div
  className="ais-InstantSearch__root"
/>
`;

exports[`autocomplete recipe3 Mentions renders without crashing 1`] = `
<div
  className="ais-InstantSearch__root"
/>
`;

exports[`autocomplete recipe3 MultiIndex renders without crashing 1`] = `
<div
  className="ais-InstantSearch__root"
>
  <div
    className="react-autosuggest__container"
  >
    <input
      aria-activedescendant={null}
      aria-autocomplete="list"
      aria-expanded={false}
      aria-haspopup={false}
      aria-owns="react-autowhatever-1"
      autoComplete="off"
      className="react-autosuggest__input"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      placeholder="Type a product"
      role="combobox"
      type="text"
      value=""
    />
    <div
      className="react-autosuggest__suggestions-container"
      id="react-autowhatever-1"
    />
  </div>
</div>
`;
